java/ql/src/semmle/code/java/security/JWTQuery.qll

```diff
@@ -0,0 +1,55 @@
+/** Provides classes to be used in queries related to JWT vulnerabilities. */
```

Comment:
I guess this file should be named `<queryFileName>Query.qll`?

Reply:
It makes more sense that way, yes. Fixed in f4bc4df, thanks!